concourse/tasks/delete_admin.yml

```diff
@@ -0,0 +1,25 @@
+---
+platform: linux
+image: docker:///governmentpaas/cf-uaac
+inputs:
+  - name: paas-cf
+  - name: cf-secrets
+  - name: cf-manifest
+  - name: admin-creds
+run:
+  path: sh
+  args:
+    - -e
+    - -c
+    - |
+      NAME=$(cat admin-creds/username)
+
+      VAL_FROM_YAML=$(pwd)/paas-cf/concourse/scripts/val_from_yaml.rb
+      UAA_ADMIN_CLIENT_PASS=$($VAL_FROM_YAML secrets.uaa_admin_client_secret cf-secrets/cf-secrets.yml)
+      UAA_ENDPOINT=$($VAL_FROM_YAML properties.uaa.url cf-manifest/cf-manifest.yml)
+
+      uaac target "${UAA_ENDPOINT}" --skip-ssl-validation
+      uaac token client get admin -s "${UAA_ADMIN_CLIENT_PASS}"
+      
+      uaac user delete "${NAME}"
+      echo "Removed user ${NAME}"
```

Comment:
I don't think we need to say that it was removed at the end, because `uaac` outputs the following on success:
```
user account successfully deleted
```

But it would be really useful to put `echo "Removing user ${NAME}"` at the top of the script so that we know username regardless of whether the delete succeeds or fails.